k8s/contiv-vpp.yaml

```diff
@@ -160,21 +183,21 @@ spec:
         - name: cni-net-dir
           hostPath:
             path: /etc/cni/net.d
-        # Used by VPP to communicate with LD_PRELOAD client, later it'll be replaced by socket
+        # LD_PRELOAD library
+        - name: vcl-ldpreload
+          hostPath:
+            path: /usr/libexec/ldpreload/vcl-ldpreload
+        # Used by VPP to communicate with LD_PRELOAD client, later it'll be replaced by socket file directory mount.
```

Comment:
Used by VPP to communicate with LD_PRELOAD clients, later it'll be replaced by a socket file directory mount.